configure.ac

```diff
@@ -4398,6 +4398,17 @@ cat <<EOF
 ***
 EOF
 fi
+
+CFLAGS_PRINT=""
+for word in $CFLAGS; do
+    CFLAGS_PRINT="$CFLAGS_PRINT $word"
+done
```

Comment:
how is this different that just printing `CFLAGS`?

Reply:
It is cleaner without all the double/triple spaces.